src/LevelEditor.cpp

```diff
@@ -0,0 +1,64 @@
+#include <fstream>
+#include <iostream>
+
+#include "RendererSetup.h"
+#include "GameObject.h"
+#include "Platform.h"
+#include "Note.h"
+#include "LevelGenerator.h"
+#include "DroppingPlatform.h"
+#include "MovingPlatform.h"
+#include "json.hpp"
+#include "LevelJson.h"
+#include "InputBindings.h"
+#include "LevelEditorMenuRenderer.h"
+#include "RendererSetup.h"
+
+#define MUSIC "music/2.wav"
+
+int main(int argc, char** argv) {
+  GLFWwindow* window = RendererSetup::InitOpenGL();
+  InputBindings::Bind(window);
+
+  LevelEditorMenuRenderer menu_renderer;
+  std::shared_ptr<MenuState> menu_state = std::make_shared<MenuState>();
+  ProgramMode program_mode = ProgramMode::MENU_SCREEN;
+
+  while (!glfwWindowShouldClose(window)) {
+    switch (program_mode) {
+      case ProgramMode::GAME_SCREEN: {
+        RendererSetup::Close(window);
+        return EXIT_SUCCESS;
+      }
+      case ProgramMode::END_SCREEN: {
+        RendererSetup::Close(window);
+        return EXIT_SUCCESS;
+      }
```

Comment:
Since there are ProgramModes that don't make sense in one of the two programs, would it be better to make two different enums, one for RhythmRunner and one for LevelEditor?